R/type-data-frame.R

```diff
@@ -28,13 +28,12 @@ new_data_frame <- function(x = list(), n = NULL, ..., class = NULL) {
 new_data_frame <- fn_inline_formals(new_data_frame, "x")
 
 
-# Light weight constructor used for tests - avoids having to repeatedly do
-# stringsAsFactors = FALSE etc. Should not be used in internal code as is
-# not a real helper as it lacks value checks.
-data_frame <- function(...) {
-  cols <- list(...)
-  new_data_frame(cols)
+data_frame <- function(...,
+                       .size = NULL,
+                       .name_repair = c("minimal", "unique", "universal", "check_unique")) {
```

Comment:
The default should be `"unique"`.

Reply:
The default in tibble is `"check_unique"`. The default in `vec_as_names()` is `"minimal"`. Do we need to worry about having so many different defaults? (Especially if we add `tibble0()` on top of this?)

Reply:
oh yeah I meant to be consistent with tibble, my mistake.